lib/winapi/winbase.h

```diff
@@ -45,6 +45,14 @@ typedef struct _INIT_ONCE
 #define INIT_ONCE_INIT_FAILED 0x00000004UL
 typedef BOOL CALLBACK PINIT_ONCE_FN(PINIT_ONCE, PVOID, PVOID *);
 
+typedef struct _CONDITION_VARIABLE {
+    INIT_ONCE initOnce;
+    PVOID eventHandles[2];
```

Comment:
nit: This could have deserved a comment about what each of these is used for.

- In my initial review-skim I was wondering why 2 of these were necessary.
- Reading the code, I still wondered a bit until I realized that one is for `WakeConditionVariable` and the other one for `WakeAllConditionVariable`.
- However, even then I was still wondering where this difference was implemented, until I remembered that Microsoft sets different auto-resets on events during creation, which eventually lead me to look for differences in `NtCreateEvent`.

Therefore a very helpful comment would have been along the lines of:

```
// eventHandles[0]: Created as SynchronizationEvent to wake a single thread
// eventHandles[1]: Created as NotificationEvent to wake all threads at once
```

(That comment answers everything I have wondered about)

Reply:
I've added the comments.